make_prg/subcommands/from_msa.py

```diff
@@ -52,95 +75,94 @@ def register_parser(subparsers):
         ),
     )
     subparser_msa.add_argument(
-        "-o",
-        "--outdir",
-        dest="output_dir",
-        action="store",
-        default=".",
-        help="Output directory. Default: %(default)s",
-    )
-    subparser_msa.add_argument(
-        "-n",
-        "--prg_name",
-        dest="prg_name",
-        action="store",
-        help="Prg file name. Default: MSA file name",
-    )
-    subparser_msa.add_argument(
-        "-S",
-        "--seqid",
-        help="Sequence identifier to use for the output sequence/PRG. Default is the file name",
-    )
-    subparser_msa.add_argument(
-        "--no_overwrite",
-        dest="no_overwrite",
+        "--output_graphs",
+        dest="output_graphs",
         action="store_true",
-        help="Do not replace an existing prg file",
+        default=False,
+        help="Outputs the recursive tree graphical representation (for development use only)",
     )
-    subparser_msa.add_argument(
-        "-O",
-        "--output-type",
-        help="p: PRG, b: Binary, g: GFA, a: All. Combinations are allowed i.e., gb: GFA and Binary. Default: %(default)s",
-        default="a",
-        type=OutputType,
-    )
-    subparser_msa.add_argument("--log", help="Path to write log to. Default is stderr")
+
     subparser_msa.set_defaults(func=run)
 
     return subparser_msa
 
 
-def run(options):
-    MSA_file = Path(options.MSA).resolve()
-    if not MSA_file.exists():
-        raise ValueError(f"File not found: {options.MSA}")
-    output_dir = Path(options.output_dir).resolve()
-    output_dir.mkdir(parents=True, exist_ok=True)
+def get_all_input_files(input_path: str) -> List[Path]:
+    input_path = Path(input_path)
+    if not input_path.exists():
+        raise FileNotFoundError(f"{input_path} does not exist")
 
-    if options.prg_name is None:
-        options.prg_name = MSA_file.stem
-    ofile_prefix = output_dir / options.prg_name
+    if input_path.is_file():
+        all_files = [input_path]
+    else:
+        all_files = [
+            Path(path).absolute() for path in input_path.iterdir() if path.is_file()
```

Comment:
```suggestion
            path.resolve() for path in input_path.iterdir() if path.is_file()
```

Don't use `Path.absolute` - https://stackoverflow.com/a/44569249/5299417

Also, `iterdir` returns the path as a Path already, so no need to convert

Reply:
Yeah, I have no idea why I converted a `Path` into `Path` again, I think before was some dir iteration using `os` that returned strings, and when I switched to `Path` iteration, forgot to remove the cast. Thanks for the tip on `Path.resolve`!